pswi/03_create.sh

```diff
@@ -28,9 +28,9 @@ echo "z/OSMF version     :" $ZOSMF_V
 # JSONs
 ADD_SWI_JSON='{"name":"'${SWI_NAME}'","system":"'${ZOSMF_SYSTEM}'","description":"ZOWE v'${VERSION}' Portable Software Instance",
 "globalzone":"'${GLOBAL_ZONE}'","targetzones":["'${TZONE}'"],"workflows":[{"name":"ZOWE Mount Workflow","description":"This workflow performs mount action of ZOWE zFS.",
-"location": {"dsname":"'${WORKFLOW_DSN}'(ZWEWRF02)"}},{"name":"ZOWE Configuration of Zowe 3.0","description":"This workflow configures Zowe v3.0.",
-"location": {"dsname":"'${WORKFLOW_DSN}'(ZWECONF)"}},{"name":"ZOWE APIML optimized Configuration","description":"This simplified workflow configures APIML for Zowe 3.0.",
-"location": {"dsname":"'${WORKFLOW_DSN}'(ZWEAMLCF)"}},{"name":"ZOWE Creation of CSR request workflow","description":"This workflow creates a certificate sign request.",
+"location": {"dsname":"'${WORKFLOW_DSN}'(ZWEWRF02)"}},{"name":"Stand-alone Zowe API ML Configuration","description":"This workflow configures only API ML for Zowe 3.0.",
+"location": {"dsname":"'${WORKFLOW_DSN}'(ZWEAMLCF)"}},{"name":"Full Zowe server-side configuration for Zowe 3.0","description":"This workflow configures all Zowe server-side components for Zowe v3.0.",
```

Comment:
these descriptions don't look right? ZWEAMLCF is APIML-only, ZWECONF is full server-side config, ZWECRECR is CSR? ZWEWRF02 I'm not familiar with

Reply:
code alignment is bit tricky there, you have to check what is within brackets {}, so PDS member (location) correspond with "label name" which is on previous line, see {"name":"Stand-alone Zowe API ML Configuration","description":"This workflow configures only API ML for Zowe 3.0.",
"location": {"dsname":"'${WORKFLOW_DSN}'(ZWEAMLCF)"}}

Reply:
line wrapping got me. thank you for clarifying 👍